packaging/hudi-cli-bundle/hudi-cli-bundle.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
Im ok with either, I think I just wanted the name to be different from the `hudi-cli.sh` that is located in the `hudi-cli` module so that people do not get confused. I think maintaining two scripts with the same name might be confusing and im wondering if we should name it specific to the platform hudi is running on? Something like `hudi-cli-emr.sh`?

Reply:
Let's not make this platform dependent.  But having a different script name makes sense.  I'm leaning towards `hudi-cli-with-bundle.sh`.

Reply:
sounds good